rfc-163/relocating-logic-from-cdn.md

```diff
@@ -0,0 +1,120 @@
+# Appendix: Proposal for relocating logic from the CDN layer to other parts of the stack
+
+Things that could be moved to WAF:
+
+- IP allowlisting on staging and production EKS[^ip-allowlist]
+- Requiring HTTP Basic auth on integration[^http-basic-1][^http-basic-2] (unless the user's IP is in the allowlist[^http-basic-allowlist])
+- IP denylisting[^ip-denylist]
+  - This functionality is currently unused (the dictionary that the denylist is read from is empty), but it exists in case we ever need to quickly block IP addresses (for example, during an incident), so we need to preserve it
```

Comment:
I would question whether we _need_ to preserve it. But definitely something to consider (post-RFC):

```suggestion
  - This functionality is currently unused (the dictionary that the denylist is read from is empty), but it exists in case we ever need to quickly block IP addresses (for example, during an incident).
```